setup.cfg

```diff
@@ -77,6 +77,11 @@ install_requires =
   # NOTE: per issue #509 0.15.34 included in debian backports
   typing-extensions; python_version < "3.8"
 
+[options.extras_require]
+docs =
+    Sphinx
+    sphinx_ansible_theme >= 0.2.2
+
```

Comment:
Extras are for *runtime* deps what enable the project do additional stuff when you run it. The right way is to use https://github.com/ansible/ansible-lint/pull/861 (which by itself belongs to a separate PR).